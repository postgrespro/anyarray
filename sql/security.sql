-- Check security CVE-2020-14350
CREATE FUNCTION rumanyarray_distance(anyarray,anyarray) RETURNS bool AS $$ SELECT false $$ LANGUAGE SQL;
CREATE EXTENSION anyarray CASCADE;
DROP FUNCTION rumanyarray_distance(anyarray,anyarray);

