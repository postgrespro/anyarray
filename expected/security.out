-- Check security CVE-2020-14350
CREATE FUNCTION rumanyarray_distance(anyarray,anyarray) RETURNS bool AS $$ SELECT false $$ LANGUAGE SQL;
CREATE EXTENSION anyarray CASCADE;
NOTICE:  installing required extension "rum"
ERROR:  function "rumanyarray_distance" already exists with same argument types
DROP FUNCTION rumanyarray_distance(anyarray,anyarray);
